xmrstak/backend/amd/amd_gpu/opencl/cryptonight.cl

```diff
@@ -30,6 +30,7 @@ R"===(
 #define cryptonight_superfast 12
 #define cryptonight_gpu 13
 #define cryptonight_conceal 14
+#define cryptonight_v8_reversewaltz 15
```

Comment:
This needs to be 17, not 15, to agree with the value in `xmrstak/backend/cryptonight.hpp`

Reply:
Okay. Fixed. Thanks, @jagerman.